packages/protocol-contracts/contracts/ZetaTokenConsumerRecommended.strategy.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT
```

Comment:
@brewmaster012 what do you think about this?
On one side, it's convenient because it allows dApp developers to rely on us on the strategy to get Zeta, if we happen to change from UniswapV3 to another swap strategy someday, they won't even realize.
On the other side, it's a new way in which we ask the developers to trust us, and we'll need to maintain it and keep it safe.
Also, if we're gonna use TSS as the address that updates the strategy on this contract, we'll need to add logic that does the update to the protocol.

The alternative is only deploying UniV2 and UniV3 contracts and:
1. Asking developers to update the strategy address in case we deploy a better alternative and they wanna use it
2. Asking them to redeploy their contracts if they don't want to make the address dynamic
3. Asking them to deploy their own strategy if they don't want to use ours